### PR TITLE
HAAR-804: access to create-role page now role protected

### PR DESCRIPTION
**Pull request by AlTurner-MOJ**

### CODE REVIEW

File: backend/middleware/authorisationMiddleware.test.ts [hunk 1]
```diff
@@ -161,6 +161,27 @@ describe('authorisationMiddleware', () => {
     })
   })
 
+  describe('create-role', () => {
+    it('should show correct page when user with correct role tries to access create-role endpoint', () => {
+      const reqCreateExternalUsers = { originalUrl: '/create-role' } as Request
```
Comment (karenmillermoj) [inline backend/middleware/authorisationMiddleware.test.ts:166]: should name of variable be something like: req/request/requestCreateRole

File: backend/middleware/authorisationMiddleware.ts [hunk 2]
```diff
@@ -17,6 +18,7 @@ const authorisationMap = {
   '/create-user': [AuthRole.CREATE_USER],
   '/delete/children/none': [AuthRole.MAINTAIN_OAUTH_USERS],
   '/create-external-user': [AuthRole.MAINTAIN_OAUTH_USERS, AuthRole.GROUP_MANAGER],
+  '/create-role': [AuthRole.ROLES_ADMIN],
```
Comment (karenmillermoj) [inline backend/middleware/authorisationMiddleware.ts:21]: We need an additional test in the UI/cypress test to ensure the page goes to the AuthError if unauthorized

File: backend/middleware/authorisationMiddleware.test.ts (hunk 1 above)
Comment (AlTurner-MOJ) [reply backend/middleware/authorisationMiddleware.test.ts]: Agree & changed

File: backend/middleware/authorisationMiddleware.ts (hunk 2 above)
Comment (AlTurner-MOJ) [reply backend/middleware/authorisationMiddleware.ts:21]: Agree and added

**Review by karenmillermoj — APPROVED** [review]
lgtm

**Pull request merged**